Makefile

```diff
@@ -27,7 +27,13 @@ latest:
 
 .PHONY:
 test-inside: build
-	docker-compose run --rm -e TRAVIS wsgi scripts/launch_inside_tests.sh
+	docker run --rm -v $$PWD/data:/data c2corg/v6_images:latest rm -rf /data/incoming/* /data/active/* /data/.minio.sys/format.json /data/.minio.sys/multipart  /data/.minio.sys/tmp
```

Comment:
What is this line for?

Reply:
If the previous run failed, this same line wasn't executed after the test and the bucket wasn't cleared.